src/os/bluestore/BlueFS.cc

```diff
@@ -2237,8 +2237,10 @@ void BlueFS::_drop_link_D(FileRef file)
     log.t.op_file_remove(file->fnode.ino);
     nodes.file_map.erase(file->fnode.ino);
     logger->set(l_bluefs_num_files, nodes.file_map.size());
-    file->deleted = true;
-
+    {
```

Comment:
funaction name to get _F substr in the name here and there?

Reply:
It should and it will.